### PR TITLE
Tidy up the `Event`-related documentation

### LINKED ISSUE #4174: Event documentation doubles up the bubble and verbose checkboxes
Each of the [events](https://textual.textualize.io/events/) in the documentation have the bubble and verbose checkboxes doubled up; once early on on the page and once again later on.

In at least one case ([`Paste`](https://textual.textualize.io/events/paste/)) they don't even agree.

### PR DESCRIPTION
**Pull request by davep**
A PR that wanders through the `Event`-related docs and tidies up some issues. Main changes are:

- Removes almost all of the in-md-file documentation for all documented events and instead relies on their docstrings.
- Adds missing docstrings where appropriate.
- Adds more see-also goodness where noticed and appropriate.
- Adds `Print` to the documented events.
- Adds `AppBlur` and `AppFocus` to the list of events now that they're more widely-used.

Fixes #4174.

### CODE REVIEW

**Comment by davep** [comment]
Taking back to draft for the moment. We've decided to go all-in on letting the events self-document (as much as is possible), so I'll go back over these gain and remove almost everything from the `md` files themselves, and also harmonise each of the events in terms of what has docstrings and what form they take.

File: src/textual/events.py [hunk 1]
```diff
@@ -563,22 +571,24 @@ class Blur(Event, bubble=False):
...
+        `FocusIn`, or when running via textual-web.
     """
 
 
 class AppBlur(Event, bubble=False):
-    """Sent when the app loses focus.
-
-    Only available when running within a terminal that supports `FocusOut`,
-    or when running via textual-web.
+    """Sent when the app gains focus.
```
Comment (TomJGooding) [inline src/textual/events.py:584]: I don't envy you working through all these! Hope you don't mind me sticking my nose in as usual, but spotted this earlier.

```suggestion
    """Sent when the app loses focus.
```

Comment (davep) [reply src/textual/events.py]: Good catch! I have *no* idea what happened there; can't for the life of me figure out what I was thinking.

**Review by willmcgugan — APPROVED** [review]
Works for me.

**Pull request merged**